app/models/recipe.rb

```diff
@@ -16,6 +17,10 @@ class Recipe < ActiveRecord::Base
 
   acts_as_taggable
 
+  def forked_from
+    Recipe.where(id: forked_from_recipe_id).first
```

Comment:
yeah this should not be a method but a association. it is the inverse of `has_many :forks`. so it should be `belongs_to :forked_from`